Dockerfile.plugins

```diff
@@ -27,6 +26,9 @@ ARG FIREHOSE_PLUGIN_BRANCH=""
 ARG CLOUDWATCH_PLUGIN_CLONE_URL=https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit.git
 ARG CLOUDWATCH_PLUGIN_TAG=""
 ARG CLOUDWATCH_PLUGIN_BRANCH=""
+ARG BUILD_TYPE="RELEASE"
+
+RUN if [ "$BUILD_TYPE"="LOCAL"  ];then go env -w GOPROXY=direct;fi
```

Comment:
The docker build for this is called here: https://github.com/aws/aws-for-fluent-bit/blob/mainline/scripts/build_plugins.sh#L177

So you need to pass the var to there or else it won't work. 

Thank you @ShelbyZ for figuring this out.

Reply:
See here as well for setting different values for a Makefile var in different targets: https://github.com/aws/aws-for-fluent-bit/blob/mainline/Makefile#L24